docs/content/concepts/dagster-pipes/aws-emr-serverless.mdx

```diff
@@ -0,0 +1,163 @@
+---
```

Comment:
If possible, could you separate the new guide content from the ruff formatting of docs_snippets?